discoverable-client/src/test/java/com/ca/mfaas/client/api/PetControllerDeleteTest.java

```diff
@@ -27,7 +27,7 @@
 
 @RunWith(SpringRunner.class)
 @WebMvcTest(controllers = {PetController.class}, secure = false)
-@Import(ApplicationConfiguration.class)
+@Import(value = {ApplicationConfiguration.class})
```

Comment:
Why did u need to change ```@Import...``` lines on the tests?

Reply:
"value" is an array of strings, so curly brackets are very legal syntax.
Remained after experimenting with import of multiple components.

Reply:
It is legal but you have only one configuration class for now. For one, you don't need brackets

Reply:
Ok, Changed the single value without "value"